### PR TITLE
Add skill challenge rules

### PR DESCRIPTION
**Pull request by LukeAngove**
Add first iteration of the skill challenge rules. It doesn't specify anything about gem movement, or interactions with cards.

### CODE REVIEW

File: Skills.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Skill/Attribute Challenges
+
+Challenges are represented by rolling dice either over, or under a given threshold for multiple attributes (elements).
+This gives the GM a lot of flexibility around how skill challenges occur. For any particular skill challenge,
+the GM picks 3 attributes (attributes may be repeated). The player must then roll a die, and get under the threshold
+for each attribute. The threshold is the 2x number of gems in the spheres matching the element in question, +1 for each gem
```
[the remark this answers is not in the document]
Comment (LukeAngove) [reply Skills.md]: Updated with latest revision. Pretty major changes.

File: Skills.md [hunk 2]
```diff
@@ -1,61 +1,80 @@
...
-This gives the GM a lot of flexibility around how skill challenges occur. For any particular skill challenge,
-the GM picks 3 attributes (attributes may be repeated). The player must then roll a die, and get under the threshold
-for each attribute. The threshold is the 2x number of gems in the spheres matching the element in question, +1 for each gem
-of the same element. The die rolled
-depends on the number of gems owned of the matching element. Thresholds can be adjusted by the GM to demonstrate difficulty.
+## Challenge Rules
+Challenges are handled by rolling three dice, associated with three elements. There is one major element, and two minor.
+A success requires rolling greater than a threshold on the major element, and a least one of the minor elements. A failure
+on only one of the minor elements results in a minor negative side effect, the details of which are left up to the GM,
+but where possible associated with the element that failed.
```
Comment (nitrogenfingers) [inline Skills.md]: Any thoughts on a major failure but two minor successes? Conditions/consequences for a glitch?

File: Skills.md [hunk 3]
```diff
@@ -1,61 +1,80 @@
...
-
-Note that the threshold can never be below 1. If characters shouldn't be able to do something, just say so, and don't bother
-with a test. It is reasonable to say, for example, "You have no experience with this; you try for a while, but fail to accomplish
-anything".
-
 For example:
 Robert wants to ask a villager for help. The GM asks how he wants to do so. Robert says that he wants to plea, pulling on the
 villagers heart strings. The GM doesn't see any physical impact from the action, and determines the Robert is trying to actively
-change the villagers mind, so he calls for an easy 'lww' (Lightning, Water, Water) test, as lightning represents change,
+change the villagers mind, so he calls for a 'Water, Lightning, Water, threshold 8' test, as lightning represents change,
```
Comment (nitrogenfingers) [inline Skills.md]: I think a description of what each element means and how they relate to the skill checks being made would be an important addition, both to provide GMs some basis for making calls on elements, and for players to assist them in their assumptions. Providing examples for a few different typical skill checks is probably a good idea too (perception, athletics, persuasion and stealth are good candidates). That probably belongs to a separate PR.

File: Skills.md [hunk 4]
```diff
@@ -1,61 +1,80 @@
...
+
+| Level           | Threshold | 50% pass    | >=75% pass  | >=90% pass  | 100% pass  |
+| --------------- | --------- | ----------- | ----------- | ----------- | ---------- |
+| Trivial         |  3        | None        | d8          | d10, 1n     | 2n         |
+| Easy            |  5        | d8          | d8, 1m      | d10, 1m, 1n | 5n         |
+| Average         |  7        | d8, 1m      | d8, 1m, 2n  | d10, 2m, 2n | 2m, 2n     |
+| Challenging     |  9        | d8, 1m, 2n  | d10, 2m     | d12, 3m, 1n | 3m, 2n     |
+| Hard            | 11        | d10, 2m, 1n | d12, 3m, 1n | None        | None       |
+| Very Hard       | 13        | d12, 3m     | None        | None        | None       |
+| Nigh Impossible | 15        | d12, 3m, 2n | None        | None        | None       |
```
Comment (nitrogenfingers) [inline Skills.md]: I think this was a good call. Raw thresholds are easier to parse than multiple modified results.

File: Skills.md (hunk 2 above)
Comment (LukeAngove) [reply Skills.md]: Not overly yet, though I think it could have some interesting ideas.

File: Skills.md (hunk 3 above)
Comment (LukeAngove) [reply Skills.md]: Agreed. I'll be working on that soon.

File: Skills.md (hunk 4 above)
Comment (LukeAngove) [reply Skills.md]: Agreed.

**Pull request merged**